src/scenes/home/mentorRequest/mentorRequest.js

```diff
@@ -111,7 +111,7 @@ class MentorRequest extends Component {
       a user selects from to set this property are fetched via API from the Airtable Skillsets
       table, which contains exact matches for the predefined skillset options on the Airtable
       Mentor Request table. */
-      this.setState({ submitError: 'There was an error requesting a mentor.' });
+      this.setState({ submitError: 'There was an error requesting a mentor: please join operation code slack workspace for assistance.' });
```

Comment:
We should also include the email for operation code tech support because a lot of people are going to be like "Well how tf do I join slack?"

Reply:
For now yes. But the issue is it's going to shift the burden of these "I haven't joined slack yet" from the mentor channel to the tech email which are arguably an improper escalation to higher tier support.

Reply:
Ok, I'll approve then, but we need a way for me to be notified when this happens so I can send out the slack invite

Reply:
The text should probably be 'Operation Code Slack organization' instead of 'operation code slack workspace'.

To clarify the intent here - do we want this error to be displayed every time there is any error in requesting a mentor? Sometimes there are issues w/ the front-end communicating with the API, or network issues, or even back-end issues where the request fails. 

If we change the message, even when that happens, the error message displayed would always be : "'There was an error requesting a mentor: please join Operation Code Slack organization for assistance."

Reply:
I feel like if you know you're a member that you would know to go on slack and ask for #help but maybe there could be a bit more generic wording?

Reply:
It makes more sense to have the backend pipe the message to the frontend. But yes for the time being we'd prefer for them to get assistance in slack. If they cannot join slack, or they aren't getting an invite then at that point they should be sending an email to the tech email.